config/runtime.exs

```diff
@@ -99,13 +99,36 @@ config :logger, Sentry.LoggerBackend,
   # Send messages like `Logger.error("error")` to Sentry
   capture_log_messages: true
 
-# Domain is the email address that mailgun is sent from
-domain = System.get_env("DOMAIN")
-# Configure Mailgun
-config :chat_api, ChatApi.Mailers.Mailgun,
-  adapter: Swoosh.Adapters.Mailgun,
-  api_key: mailgun_api_key,
-  domain: domain
+case mailer_adapter do
+  "Swoosh.Adapters.Mailgun" ->
+    config :chat_api, ChatApi.Mailers,
+      adapter: Swoosh.Adapters.Mailgun,
+      api_key: System.get_env("MAILGUN_API_KEY"),
+      # Domain is the email address that mailgun is sent from
+      domain: System.get_env("DOMAIN")
+
+  "Swoosh.Adapters.SMTP" ->
+    config :chat_api, ChatApi.Mailers,
+      adapter: Swoosh.Adapters.SMTP,
+      relay: System.get_env("SMTP_HOST_ADDR", "mail"),
+      port: System.get_env("SMTP_HOST_PORT", "25"),
+      username: System.get_env("SMTP_USER_NAME"),
+      password: System.get_env("SMTP_USER_PWD"),
+      ssl: System.get_env("SMTP_HOST_SSL_ENABLED") || false,
+      tls: :if_available,
+      retries: System.get_env("SMTP_RETRIES") || 2,
+      no_mx_lookups: System.get_env("SMTP_MX_LOOKUPS_ENABLED") || true
+
+  "Swoosh.Adapters.Local" ->
+    config :swoosh,
+      serve_mailbox: System.get_env("LOCAL_SERVE_MAILBOX", "true") == "true",
```

Comment:
Had to convert this to boolean & the `preview_port` below to int. The local adapter does not accept strings unlike the SMTP adapter